perma_web/perma/views/link_management.py

```diff
@@ -58,7 +58,8 @@ def create_link(request):
         'links_remaining': links_remaining,
         'folder': folder,
         'suppress_reminder': suppress_reminder,
-        'max_size': max_size
+        'max_size': max_size,
+        'org_user': request.user.is_organization_user
```

Comment:
Total nitpick: `request` is always in the template context, so I would tend to just use `{{ request.user.is_organization_user }}`.